Apps/phgsgmail/gsgmail_get_email.html

```diff
@@ -0,0 +1,135 @@
+{% extends 'widgets/widget_template.html' %}
+{% load custom_template %}
+
+{% block custom_title_prop %}{% if title_logo %}style="background-size: auto 60%; background-position: 50%; background-repeat: no-repeat; background-image: url('/app_resource/{{ title_logo }}');"{% endif %}{% endblock %}
+{% block title1 %}{{ title1 }}{% endblock %}
+{% block title2 %}{{ title2 }}{% endblock %}
+{% block custom_tools %}
+{% endblock %}
+
+{% block widget_content %} <!-- Main Start Block -->
+
+<!-- File: gsgmail_get_email.html
+  Copyright (c) 2017-2021 Splunk Inc.
+
+  Licensed under Apache 2.0 (https://www.apache.org/licenses/LICENSE-2.0.txt)
+-->
+
+<style>
+
+    a:hover {
+        text-decoration:underline;
+    }
+
+    .wf-table-vertical {
+    width: initial;
+    font-size: 12px;
+    }
+
+    .wf-table-vertical td{
+    padding: 8px 10px;
+    border: 1px solid;
+    }
+
+    .wf-table-vertical tr td:first-child {
+    font-weight: bold;
+    }
+
+    .wf-table-horizontal {
+        width: initial;
+        border: 1px solid;
+        font-size: 12px;
+    }
+
+    .wf-table-horizontal th {
+        text-align: center;
+        border: 1px solid;
+        text-transform: uppercase;
+        font-weight: normal;
+        padding: 5px;
+    }
+
+    .wf-table-horizontal td {
+        border: 1px solid;
+        padding: 5px;
+        padding-left: 4px;
+    }
+
+    .wf-h3-style {
+        font-size : 20px
+    }
+
+    .wf-h4-style {
+        font-size : 16px
+    }
+
+</style>
+<div class="gmail-get-email" style="overflow: auto; width: 100%; height: 100%; padding-left:10px; padding-right:10px"> <!-- Main Div -->
+    {% for result in results %} <!-- loop for each result -->
+    <br>
+
+    <!------------------- For each Result ---------------------->
+    {% if not result.data %}
+    <h4 class="wf-h4-style">No messages found</h4>
+
+    {% else %}
+    <table class="wf-table-horizontal datatable">
```

Comment:
Nice! I'm wondering how this looks like. Could you share the link to your Test machine?
Also, seeing the class `datatable` above, I'm wondering if this looks different than our default table format.